sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1014,12 +1014,12 @@ object SQLConf {
       "`parquet.compression` is specified in the table-specific options/properties, the " +
       "precedence would be `compression`, `parquet.compression`, " +
       "`spark.sql.parquet.compression.codec`. Acceptable values include: none, uncompressed, " +
-      "snappy, gzip, lzo, brotli, lz4, lz4raw, zstd.")
+      "snappy, gzip, lzo, brotli, lz4, lz4_raw, zstd.")
```

Comment:
What is the suffix of the file after changing to `LZ4_RAW`?

Reply:
Before this PR:
`part-00000-fc07d464-03b2-42d6-adc1-68a3adca1752.c000.lz4raw.parquet`
After this PR:
`part-00000-07244014-f31a-4097-8878-dd3630e721ce.c000.lz4raw.parquet`

Reply:
Since there is no change at file name layer, it's good.